docs/tutorials/quickstart.md

```diff
@@ -4,15 +4,21 @@ Blueapi acts as a worker that can run bluesky plans against devices for a specif
 laboratory setup. It can control devices to collect data and export events to tell
 downstream services about the data it has collected.
 
-## Start RabbitMQ
+## Start Services
 
-The worker requires a running instance of RabbitMQ. The easiest way to start it is
- to execute the provided script:
+The worker requires other infrastructure services running- these are captured in a [compose-spec](https://github.com/compose-spec/compose-spec/blob/main/spec.md) file: `.devcontainer/compose.yml`, and may later be started as part of building the devcontainer when [compatibility with podman is improved](https://github.com/devcontainers/cli/issues/863).
 
+To run the services:
 ```
-    src/script/start_rabbitmq.sh
+    # with docker
+    docker compose -f .devcontainer/compose.yml up
+    # or with podman-compose
+    podman-compose -f .devcontainer/compose.yml up
```

Comment:
Should: Although we gave up on autostarting these services in the devcontainer, can we make `docker-compose` available in the devcontainer so the developer can do it manually? Otherwise this whole section doesn't apply if you're using a devcontainer.